script.js

```diff
@@ -0,0 +1,55 @@
+let library = localStorage.getItem('library') == null
+  ? []
+  : JSON.parse(localStorage.getItem('library'));
+
+function saveLocalStorage() {
+  localStorage.setItem('library', JSON.stringify(library));
+}
+
+function createTR(html) {
+  const div = document.createElement('table');
+  div.innerHTML = html;
+  return div.querySelector('tr');
+}
+
+function paintBooks() {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+  library.forEach((Book) => {
+    const newBookElement = createTR(`
+      <tr>
+          <td>${Book.title}</td>
+          <td>${Book.author}</td>
+          <td><button>Remove Book</button></td>
+      </tr>
+    `);
+    newBookElement.querySelector('button').addEventListener('click', () => {
+      newBookElement.remove();
+      library = library.filter((bookArr) => Book !== bookArr);
+      saveLocalStorage();
+    });
+    bookList.appendChild(newBookElement);
+  });
+}
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function add(newBook) {
+  library.push(newBook);
```

Comment:
- [ ] [OPTIONAL] Consider favouring the use of array methods that don't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.